mlir/test/Dialect/Vector/linearize.mlir

```diff
@@ -246,6 +257,16 @@ func.func @test_vector_extract(%arg0: vector<2x8x2xf32>) -> vector<8x2xf32> {
   return %0 : vector<8x2xf32>
 }
 
+// ALL-LABEL:   func.func @test_vector_extract_scalable(
+// ALL-SAME:    %[[VAL_0:.*]]: vector<2x[2]xf32>) -> f32 {
+func.func @test_vector_extract_scalable(%arg1: vector<2x[2]xf32>) -> f32 {
```

Comment:
Why are types inside this tens and `@test_vector_extract` different? Is this in any way significant?

Reply:
Changed the types for scalable tests to closely resemble normal ones.